app/workers/zendesk_ticket_worker.rb

```diff
@@ -1,11 +1,20 @@
 class ZendeskTicketWorker
   include Sidekiq::Worker
-  
+
   def perform(ticket_options)
     if GDS_ZENDESK_CLIENT.users.suspended?(ticket_options["requester"]["email"])
       $statsd.increment("#{::STATSD_PREFIX}.report_a_problem.submission_from_suspended_user")
     else
-      GDS_ZENDESK_CLIENT.ticket.create!(HashWithIndifferentAccess.new(ticket_options))
+      begin
+        create_ticket(ticket_options)
+      rescue ZendeskAPI::Error::NetworkError => e
+        raise unless e.response.status == 409
```

Comment:
I think it may also be worth adding a comment in the code to explain what makes the 409 status code so special in this case, but that's subjective...